src/Education/IEducation.cs

```diff
@@ -68,34 +77,43 @@ public void ShowRuleHelp(SonarCompositeRuleId ruleId)
         {
             var ruleInfo = ruleMetadataProvider.GetRuleInfo(ruleId);
 
-            if (ruleInfo == null)
-            {
-                logger.WriteLine(Resources.Education_NoRuleInfo, ruleId.ErrorListErrorCode);
-                return;
-            }
-
             threadHandling.RunOnUIThread(() =>
             {
-                // Lazily fetch the tool window from a UI thread
-                if (ruleHelpToolWindow == null)
+                if (ruleInfo == null)
                 {
-                    ruleHelpToolWindow = toolWindowService.GetToolWindow<RuleHelpToolWindow, IRuleHelpToolWindow>();                    
+                    showRuleInBrowser.ShowRuleDescription(ruleId);
```

Comment:
If we can't find the rule info locally, fallback on showing it in the IDE.

This is a better general behaviour that doing nothing. 
More importantly, it means that `IEducation` will now also be able to handle requests for taint rules. We don't have the info locally, but at least it will be shown in the browser.